### ISSUE TITLE
Inconsistent statements regarding the invalid parameter handler

### ISSUE THREAD

**Issue by BillMedland** [MicrosoftDocs/cpp-docs#4367]
tl;dr. The 'Return Value' section states, **_without caveat_**, that "If the storage ... exceeds ..., the invalid parameter handler is invoked,". The 'Remarks' section contains an apparent caveat in the case of the count being _TRUNCATE, although it does not state that the handler is not called.

Thanks for the detailed documentation. I have always admired the tightness and correctness of Microsoft's API documentation and so this one surprised me.
The snprintf_s documentation points to this page with a comment about the superior handling on buffer overflow, and so it is particularly confusing that the feature is not clarified.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 1caceb0b-ffbf-38f7-ded1-cb1e7857e2af
* Version Independent ID: 789a9113-5df5-0b5e-6b7d-123c64ef61f4
* Content: [_snprintf_s, _snprintf_s_l, _snwprintf_s, _snwprintf_s_l](https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/snprintf-s-snprintf-s-l-snwprintf-s-snwprintf-s-l?view=msvc-170)
* Content Source: [docs/c-runtime-library/reference/snprintf-s-snprintf-s-l-snwprintf-s-snwprintf-s-l.md](https://github.com/MicrosoftDocs/cpp-docs/blob/main/docs/c-runtime-library/reference/snprintf-s-snprintf-s-l-snwprintf-s-snwprintf-s-l.md)
* Product: **visual-cpp**
* Technology: **cpp-ucrt**
* GitHub Login: @TylerMSFT
* Microsoft Alias: **twhitney**

**Comment by TylerMSFT** [comment]
I recently added a behavior table that I think will address your concern.
